### DISCUSSION TITLE
Can I use the sma S3  bucket to backup multiple pvc ?

### DISCUSSION THREAD

**Discussion by mehdi-wsc** [backube/volsync#164] [Q&A]
Hello,
I'm new with volsync, I'm testing it for my disaster recovery plan, I have OpenShift clusters and in all my namespaces where I have more than one pvc, the backup doesn't work, the job keeps on pending state.
Can I use the same s3 bucket to backup multiple pvc ? if yes, how can I achieve this ? 
Regards,

**Comment by JohnStrunk** [comment]
In case someone else is having this problem, I believe the solution for rclone is here: https://github.com/backube/volsync/issues/162#issuecomment-1068068501
Credit: @tesshuflower
